examples/md-flexible/tests/ThermostatTest.cpp

```diff
@@ -57,6 +57,9 @@ TEST_F(ThermostatTest, BrownianMotionTest_InitV1) {
 }
 
 TEST_F(ThermostatTest, MultiComponentTest) {
+#if defined(AUTOPAS_INCLUDE_MPI)
+  EXPECT_EQ(true, true);
```

Comment:
If a test does not apply it is better to skip it.
```suggestion
    GTEST_SKIP() << "MulitComponent test is only feasible without MPI";
```
But actually, I'm not really sure why? Can you explain this and add a comment about this in the code? Is it because the communication logic is missing in the test? We could just add this...

Reply:
You are right, this preprocessor switch is not required. I implemented it when the memory leak was still present which caused the tests to fail. I removed the swtiches from the tests.